templates/shared/forms/interactive/automotive.html

```diff
@@ -0,0 +1,78 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <div class="js-pagination js-pagination--1" style="padding-top:2rem;">
+      <div class="u-sv2 js-formfield">
+        <h3 class="p-heading--4">What can we help you with?</h3>
+        <textarea id="open-question" name="open-question" aria-label="What can we help you with?" rows="3"></textarea>
```

Comment:
Should we wrap this in a a `col-8` or something similar instead? It looks a bit wider than usual like this

Reply:
Yes, good catch!

Reply:
> Both of @mtruj013's comments need addressing - we need to include a close / 'X' button at the top right like we do with other forms, it's good for the user to clearly see that there's a way to close it. I think what's been suggested in the copy doc, to have "Contact us" as a title, is a good idea.

Sorry, I didn't push up my changes. You should be able to see them now.